hack/run-origin-tests.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
+
+skopeo_workdir=$(mktemp -d)
+function finish_skopeo {
+  rm -rf ${skopeo_workdir}
+}
+trap finish_skopeo EXIT
+
+function get_ose_tests_binary {
+    # When running in a pod, we can't directly use the image.
+    # For this reason, we download the image and fetch the binary from the right layer
+    skopeo copy docker://"$ORIGIN_TESTS_IMAGE" oci:${skopeo_workdir}
+    echo "Fetching openshift-tests binary from $ORIGIN_TESTS_IMAGE"
+    for layer in ${skopeo_workdir}/blobs/sha256/*; do
+        echo "layer=${layer}"
+        set +e
+        testsbin=$(tar -t -f "$layer" | grep openshift-tests)
+        set -e
+        if [[ $testsbin ]]; then
+            echo "Found $testsbin on $layer"
+            tar xfv "$layer" "$testsbin"
+            mv "$testsbin" _cache/tools/openshift-tests
+            chmod +x _cache/tools/openshift-tests
+            rm -rf ${skopeo_workdir}
+            break
+        fi
+    done
+}
+
+CONTAINER_MGMT_CLI="${CONTAINER_MGMT_CLI:-docker}"
+TESTS_REPORTS_PATH="${TESTS_REPORTS_PATH:-/tmp/artifacts/}"
+
+ORIGIN_TESTS_IMAGE="${ORIGIN_TESTS_IMAGE:-quay.io/openshift/origin-tests:4.8}"
```

Comment:
can we take the version from common.sh?
I think it worthwhile changing this across the board (where possible)

Reply:
it'll make it (much) easier and less error prone to update the version

Reply:
do you mean to add ORIGIN_TESTS_IMAGE to common.sh?

Reply:
no, just something like
`OCP_VERSION=4.8`
and then use it here and everywhere else that need a version

Reply:
Great idea!
I'd leave this for a separate PR though, wdyt?
Also, there are a couple of places where it would not immediate to use the env variable (although we could set it in the image). 

A couple that come into my mind are: 
- the json containing the mirroring images
- the default tag used by the tests in go code

I'd track this somewhere

Reply:
let's introduce the variable here
and track the work to convert other places in a different PR.
but at least we'll have the system to do that inplace.

Reply:
agree